Project_source/app/src/main/java/android/clicker/school_live_simulator/NullComputerCourseState.kt

```diff
@@ -0,0 +1,20 @@
+package android.clicker.school_live_simulator
+
+open class NullComputerCourseState: ComputerCourseState {
+    override price: Int = 0
```

Comment:
@Ferrociraptus, There is no information in ClickerUPD about prices for courses...

Reply:
What to do with prices of courses? Inform me please when they will be added in ClickerUPD and I'll add them.

Reply:
> What to do with prices of courses? Inform me please when they will be added in ClickerUPD and I'll add them.

You can now give them zero values and just add new issue like Andrey did to add them in the future

Reply:
Fixed